modules/react/layout/lib/utils/gridItem.ts

```diff
@@ -0,0 +1,72 @@
+import {Property} from 'csstype';
+
+/** style props to for grid item properties */
+export type GridItemStyleProps = {
+  // Child Types
+  /**
+   * sets `display` property
+   * @default 'grid'
```

Comment:
does the child still need display: grid ?

Reply:
This was missed, no it doesn't.